build.sh

```diff
@@ -17,6 +17,8 @@ cp src/styles/icons.css contemplating-action/src/styles/icons.css
 cp src/styles/theme.scss contemplating-action/src/styles/theme.scss
 rm -rf contemplating-action/src/styles/fonts
 cp src/styles/fonts contemplating-action/src/styles/
+rm -rf contemplating-action/src/assets/story_images
+cp src/assets/story_images contemplating-action/src/assets/
```

Comment:
I think this and the styles one (line 19) might need a `-r`

Reply:
The placeholder.png and thumb_placeholder.png could be ignored and sourced from core - but I don't think it matters.

Happy to leave it.